node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -287,6 +289,27 @@ func (r OsDeviceConnectivityHelperScsiGeneric) RemovePhysicalDevice(sysDevices [
 	return nil
 }
 
+func (r OsDeviceConnectivityHelperScsiGeneric) ValidateLun(lun int, sysDevices []string) error {
+	logger.Debugf("Validating lun {%v} on devices: {%v}", lun, sysDevices)
+	for _, sysDevice := range sysDevices {
+		sysDeviceParts := strings.Split(sysDevice, "/")
+		device := sysDeviceParts[len(sysDeviceParts)-1]
+
+		symbolicLinkPath := fmt.Sprintf(SysDeviceSoftLinkToLun, device)
```

Comment:
choose either one of the symbolic/soft terms and stick with it

Reply:
done